Apps/Sandcastle/gallery/CZML Polygon - Intervals, Availability.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
```

Comment:
@tiffanylu I only get the flicker on Windows.  Let's try the simple approach I mentioned above:

> Try replacing dynamicPolygon with simple geometry, e.g., a triangle on the surface with just three points, and we'll see if there is any issue.

If there's still an artifact, we can demonstrate time intervals with billboards or labels.